### ISSUE TITLE
Dynamic values update issue

### ISSUE THREAD

**Issue by oliviertassinari** [cssinjs/jss#1118]
__Expected behavior:__
Dynamic values should update.

__Describe the bug:__
Dynamic values don't update.

__Codesandbox link:__ 
https://codesandbox.io/s/material-demo-xivoe

```tsx
const styles = {
  root: props => ({
    "@media print": {
      display: props.print
    }
  })
};
```

__Versions (please complete the following information):__
 - jss: 10.0.0-alpha.17
 - Browser [e.g. chrome, safari]: Chrome 74
 - OS [e.g. Windows, macOS]: macOS
 - Related issue: https://github.com/mui-org/material-ui/issues/16135

**Comment by kof** [comment]
This specifically happens with media queries. I can see that instead of updating the media query, it creates a new empty one.

cc @HenriBeck it might be related to the issue we have seen with styled-system and duplicated media queries.

**Comment by HenriBeck** [comment]
yeah, basically we would need to remove the old one.  I think in general there is an issue with creating nested rules inside dynamic rules

**Comment by bardouni** [comment]
Hi
I have the same issue
when I log the sheet output using (sheet.toString) it shows a valid value : 
![image](https://user-images.githubusercontent.com/11160251/61231290-c5753e80-a723-11e9-9a4a-8b09a78f9b56.png)
but when I check the element's css from the inspector, it dosen't show the media query rules

**Comment by HenriBeck** [comment]
@bardouni can you provide a codesandbox so we can check your code?

Usually, there is an alternative syntax where the issue doesn't exist

**Comment by HenriBeck** [comment]
This also might be fixed with #1144

**Comment by kof** [comment]
fixed in https://github.com/cssinjs/jss/pull/1242, releasing in 10.0.1
